<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Ishan-Macs-MacBook-Pro.local" name="com.framework.TestCases.Scenario4_VerifyMoreProduct" tests="1" failures="1" timestamp="18 Feb 2018 08:33:45 GMT" time="9.342" errors="0">
  <testcase name="Scenario4" time="9.342" classname="com.framework.TestCases.Scenario4_VerifyMoreProduct">
    <failure type="java.lang.AssertionError" message="Test Scenario failed at Step --&gt;&#039; User verifies more Products on Page after scroll &#039;. Error is: java.lang.Exception: More Products didn&#039;t loaded after scroll">
      <![CDATA[java.lang.AssertionError: Test Scenario failed at Step -->' User verifies more Products on Page after scroll '. Error is: java.lang.Exception: More Products didn't loaded after scroll
	at org.testng.Assert.fail(Assert.java:94)
	at com.framework.TestCases.Scenario4_VerifyMoreProduct.Scenario4(Scenario4_VerifyMoreProduct.java:72)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:663)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:849)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1157)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1200)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1125)
	at org.testng.TestNG.run(TestNG.java:1033)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </failure>
  </testcase> <!-- Scenario4 -->
</testsuite> <!-- com.framework.TestCases.Scenario4_VerifyMoreProduct -->
